package.json

```diff
@@ -30,18 +30,20 @@
     "rebuild-docker-image": "env bash ./scripts/rebuild-docker-image.sh",
     "gzip-and-upload": "env bash ./scripts/gzip-and-upload.sh",
     "build-docs": "echo 'see ./docs-src/README.md'",
-    "test": "jest",
+    "test": "jest test/*.js",
```

Comment:
`npm test` was running the `test/integration` folder, and `smoke-test/`, which was causing errors (because the puppeteer variables aren't set). I think we just want to run the unit tests with this command.

Reply:
👍